### DISCUSSION TITLE
PyCon India 2021: Logo Typography & Website

### DISCUSSION THREAD

**Discussion by sayanchowdhury** [pythonindia/inpycon2021-tasks#23] [Ideas]
Webchirpy team has sent the typography for the logo and a mock website design. Please review the items, vote the options and comment on your views. 

**_Deadline to put your thoughts: March 25, 7PM IST._** 

**Website Mockup:** https://app.evrybo.com/share/page/34327/458016/YOaGUh1austQeTdwEmFE

**Typography** 👇🏽👇🏽👇🏽

**Comment by sayanchowdhury** [comment]
1.
![v1](https://user-images.githubusercontent.com/500628/112263852-f7c79500-8c95-11eb-9058-88fe080fc471.jpg)

**Comment by sayanchowdhury** [comment]
2.
![v2](https://user-images.githubusercontent.com/500628/112263838-f4340e00-8c95-11eb-8d24-b6d27cf19584.jpg)

**Reply by inovizz** [reply]
The colour combination is nice & bright and it would look good in most cases. Hence the vote.

**Reply by SaptakS** [reply]
I like this one, without the red dotted border (I mean why?). And the year, if they could use the lighter purple in the snake instead of the lighter purple

**Reply by iamaravindsekar** [reply]
This one looks better, What if the purple color wifi symbol comes on top of one of the characters instead of that red line, will it make sense? also looking at the design of the potential webpage, I feel that this one matches perfectly

**Reply by SaptakS** [reply]
I personally don't like too much clip art on typography, but I would be fine with or without the wifi symbol on top of maybe the "I" characters

**Reply by sayanchowdhury** [reply]
Please check this comment https://github.com/pythonindia/inpycon2021-tasks/discussions/23#discussioncomment-542433

**Comment by sayanchowdhury** [comment]
3.
![v3](https://user-images.githubusercontent.com/500628/112263854-f8f8c200-8c95-11eb-8ac2-a7992c11978d.jpg)

**Reply by abhiramr** [reply]
I really like this one.

**Reply by SaptakS** [reply]
I like the typography on this one, but not the yellow background behind 2021.

**Reply by iamaravindsekar** [reply]
This is good, but the typography doesn't go in sync with the elements inside the logo, the illustrations are not having so much of sharp edges, but the text has. Also the electronic circuit design on 2021 doesn't feels obvious with the theme of our main logo.

**Comment by sayanchowdhury** [comment]
4.
![v4](https://user-images.githubusercontent.com/500628/112263862-fac28580-8c95-11eb-938f-9f9b65a7873b.jpg)

**Reply by iamaravindsekar** [reply]
I feel that this Might look better if the text color is yellow that goes matching with the logo and the typography looks little casual and friendly without those sharp edges. Maybe the same font family as 2021 or something much better. The elements on the font like that cam,mic,phone, speaker blended into the text and the way 2021 is stacked makes this look unique

**Comment by sayanchowdhury** [comment]
5.
![v5](https://user-images.githubusercontent.com/500628/112263865-fc8c4900-8c95-11eb-868f-54451e2053b9.jpg)

**Comment by sayanchowdhury** [comment]
6.
![v6](https://user-images.githubusercontent.com/500628/112263868-fd24df80-8c95-11eb-82b0-bbcf2525bcc8.jpg)

**Comment by sayanchowdhury** [comment]
7.
![v7](https://user-images.githubusercontent.com/500628/112263874-fe560c80-8c95-11eb-99e7-89530bf69345.jpg)

**Reply by SaptakS** [reply]
strictly against this "comic" style typography

**Comment by raukadah** [comment]
@sayanchowdhury In the website, can we also include BOF/sessions schedule ? It would be good to have a volunteers page for this year. https://github.com/pythonindia/inpycon2021-tasks/issues/21

**Reply by rayandas** [reply]
> @sayanchowdhury In the website, can we also include BOF/sessions schedule ? It would be good to have a volunteers page for this year. #21

Volunteers page as in? a page with every volunteer's profile? @raukadah

**Reply by raukadah** [reply]
Something like this: https://in.pycon.org/2015/team.html

**Reply by anistark** [reply]
Also, need a section for outreach partners.

**Reply by raukadah** [reply]
> Also, need a section for outreach partners.

May be sponsors section can be replaced with outreach partners.

**Reply by anistark** [reply]
Let them come back with appropriate design?

**Comment by realslimshanky** [comment]
Assuming we're only reviewing the design, I would like to make just one suggestion. The design can be expanded for different viewports, if you look at the image below, that's what it looks like on my computer. It would be great if we can fill the space on either side since our theme is more on the dark side and the empty space is lighter. 

<img width="1680" alt="Screenshot 2021-03-25 at 12 38 51 PM" src="https://user-images.githubusercontent.com/10004530/112432541-0f228300-8d67-11eb-998f-c8fd5e27a586.png">

**Comment by sayanchowdhury** [comment]
Folks, As per the comments, Webchirpy has updated the Typography for v2. Please review and give your "Ack"

![image](https://user-images.githubusercontent.com/500628/112800444-478acf80-908d-11eb-9560-edb54709a992.png)

**Reply by SaptakS** [reply]
Looks good

**Reply by iamaravindsekar** [reply]
Nice, looks cool

**Comment by SaptakS** [comment]
Can we ask them to replace the sponsor section with a communities section? cc: @sayanchowdhury @raukadah
